NetworkFetcher:121 NullPointerException

Caused by: java.lang.NullPointerException
        at com.airbnb.lottie.network.NetworkFetcher.fetchFromNetworkInternal(NetworkFetcher.java:121)
        at com.airbnb.lottie.network.NetworkFetcher.fetchFromNetwork(NetworkFetcher.java:93)
        at com.airbnb.lottie.network.NetworkFetcher.fetchSync(NetworkFetcher.java:62)
        at com.airbnb.lottie.network.NetworkFetcher$1.call(NetworkFetcher.java:49)
        at com.airbnb.lottie.network.NetworkFetcher$1.call(NetworkFetcher.java:47)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
        at java.lang.Thread.run(Thread.java:841)

--- a/lottie/src/main/java/com/airbnb/lottie/network/NetworkFetcher.java
+++ b/lottie/src/main/java/com/airbnb/lottie/network/NetworkFetcher.java
@@ -134,7 +134,7 @@ public class NetworkFetcher {
   private LottieResult<LottieComposition> getResultFromConnection(HttpURLConnection connection) throws IOException {
     File file;
     FileExtension extension;
-    LottieResult<LottieComposition> result;
+    LottieResult<LottieComposition> result = null;
     String contentType = connection.getContentType();
     if (contentType == null) {
       // Assume JSON for best effort parsing. If it fails, it will just deliver the parse exception

--- a/lottie/src/main/java/com/airbnb/lottie/network/NetworkFetcher.java
+++ b/lottie/src/main/java/com/airbnb/lottie/network/NetworkFetcher.java
@@ -134,8 +134,14 @@ public class NetworkFetcher {
   private LottieResult<LottieComposition> getResultFromConnection(HttpURLConnection connection) throws IOException {
     File file;
     FileExtension extension;
-    LottieResult<LottieComposition> result = null;
-    switch (connection.getContentType()) {
+    LottieResult<LottieComposition> result;
+    String contentType = connection.getContentType();
+    if (contentType == null) {
+      // Assume JSON for best effort parsing. If it fails, it will just deliver the parse exception
+      // in the result which is more useful than failing here.
+      contentType = "application/json";
+    }
+    switch (contentType) {
       case "application/zip":
         Logger.debug("Handling zip response.");
         extension = FileExtension.ZIP;

